packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -69,25 +69,25 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
     let overflowMenuId = this._id + '-overflow';
 
     return (
-      <div className={ css('ms-Breadcrumb', className) } ref='renderingArea'>
+      <div className={ css('ms-Breadcrumb', className, styles.root) } ref='renderingArea'>
```

Comment:
@mdahamiwal oops realized I didn't answer clearly. Yes, we will keep them, for backwards compatibility, but we are discussing better approaches to customization.

There are 2 things we're considering:

1. A more consistent way to pass in class names that target specific elements. E.g., button has root button, an icon, a label element, and a description. We could have something like a class names map:

```tsx
<button classNames={ {
   root: 'myButton',
   label: 'myButton-label',
   isSelected: 'isSelected'
}} /> 
```

2. A javascript css approach, like Glamor or Aphrodite, where classes are generated at runtime, and rather than passing in classnames, you pass in style objects like you might with inline styles. This would perhaps be a bit safer as we could do runtime processing on the rules like auto prefixing, rtl transforms, and theming detokenization. With class names alone, the customer needs to manage theming and rtl stuff on their own. Plus it hides the actual styles from the code, so there is no way for us to translate customer desire into what makes sense in the current component.

Of course we could combine these two and document the pros and cons and let customers choose what works for them. Sometimes choice creates confusion tho.